cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -408,7 +410,8 @@ private RejectedExecutionHandler createAsyncRejectedExecutionHandler() {
       @Override
       public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
         if (!executor.isShutdown()) {
-          metricsCollector.increment("collect.async.reject", 1);
+          metricsCollector.childCollector(Constants.Metrics.Tag.NAMESPACE, Constants.SYSTEM_NAMESPACE)
```

Comment:
It's ok in this case as it only happens when the stream service is shutting down.